sections/featured-product.liquid

```diff
@@ -282,35 +282,78 @@
                         assign current_qty_for_volume_pricing = cart_qty | plus: product.selected_or_first_available_variant.quantity_rule.increment
                       endif
                     -%}
-                    <price-per-item
-                      class="no-js-hidden"
-                      id="Price-Per-Item-{{ section.id }}"
-                      data-section-id="{{ section.id }}"
-                      data-variant-id="{{ product.selected_or_first_available_variant.id }}"
-                    >
-                      {%- if product.quantity_price_breaks_configured? -%}
-                        <div class="price-per-item">
-                          {%- if current_qty_for_volume_pricing < volume_pricing_array.last.minimum_quantity -%}
-                            {%- assign variant_price = product.selected_or_first_available_variant.price
-                              | money_with_currency
-                            -%}
-                            <span class="price-per-item--current">
-                              {{- 'products.product.volume_pricing.price_at_each' | t: price: variant_price -}}
-                            </span>
-                          {%- else -%}
-                            {%- for price_break in volume_pricing_array -%}
-                              {%- if current_qty_for_volume_pricing >= price_break.minimum_quantity -%}
-                                {%- assign price_break_price = price_break.price | money_with_currency -%}
-                                <span class="price-per-item--current">
-                                  {{- 'products.product.volume_pricing.price_at_each' | t: price: price_break_price -}}
-                                </span>
-                                {%- break -%}
-                              {%- endif -%}
-                            {%- endfor -%}
-                          {%- endif -%}
+                    {%- if product.quantity_price_breaks_configured? -%}
+                      {%- if product.selected_or_first_available_variant.quantity_price_breaks.size > 0 -%}
```

Comment:
Sorry i think my previous suggestion had a bug because it seems the variant change logic is only replacing the innerHTML portion of the element.

We can replace this in `global.js`:
```JS
pricePerItemDestination.innerHTML = pricePerItemSource.innerHTML;
```
With:
```
pricePerItemDestination.replaceWith(pricePerItemSource);
```


To be safe, we can also revert [this] (https://github.com/Shopify/dawn/pull/2851/commits/6c81fb718021c93c6b0715371b28d919e1969c2b), if we don't want to introduce any regressions.